uTensor/core/vm.hpp

```diff
@@ -2,7 +2,6 @@
 #define UTENSOR_VM
 #include <vector>
 #include <iostream>
-#include "mbed.h"
```

Comment:
The only dependency here is the file system. Making uTensor platform independent is a good idea. But, we probably need a HAL layer here as remove the mbed.h

Reply:
@neil-tan It both compiles and runs on OSX and Linux right now + original mbed. I don't thing we actually need a PAL layer because as long as we adhere to using only C/C++ provided bindings. The only other reason I can see a PAL layer being useful is if we add thread support, but this doesnt make too much sense on Cortex M.

Reply:
Ok, as long as it builds, I'm all for it.